gui/src-tauri/src/main.rs

```diff
@@ -1,16 +1,29 @@
 #![cfg_attr(
-	all(not(debug_assertions), target_os = "windows"),
+	all(not(debug_assertions), windows),
 	windows_subsystem = "windows"
 )]
 
 use clap::Parser;
 use clap_verbosity_flag::{InfoLevel, Verbosity};
+use native_dialog::MessageDialog;
+use native_dialog::MessageType;
+use rand::seq::SliceRandom;
+use rand::thread_rng;
 use std::env;
+use std::panic;
 use std::path::PathBuf;
 use tauri::api::clap;
 use tauri::api::process::Command;
 use tauri::Manager;
 
+static POSSIBLE_TITLES: &[&str] = &[
```

Comment:
```suggestion
const POSSIBLE_TITLES: &[&str] = &[
```

Reply:
const for arrays isnt a smart idea, if we ever use that variable more than one time it will be making 2 arrays.
More info in here: https://doc.rust-lang.org/std/keyword.const.html#compile-time-constants

Reply:
TIL